tests/TestFileWriterRotationScenarios/tst_FileWriterRotations.cpp

```diff
@@ -623,60 +634,66 @@ void FileWriterRotations::testRotateForTimePolicyAndSizeHigherNewer(int compress
 
     overrideLastWrittenDateTime(dt);
     setTestingCurrentDateTime(dt);
-    setOutputFile("log.txt");
 
-    QVERIFY(QFileInfo::exists(filenames[0]));
+    //in this test we use a subfolder
+    QDir::current().mkdir("test_subfolder");
+    QString path = QDir::currentPath() + "/test_subfolder";
+    setOutputFile(path + "log.txt");
 
-    writeToFile(filenames[0], 2);
+    QVERIFY(QFileInfo::exists(getFileWithFullPath(filenames[0], path)));
+
+    writeToFile(path + filenames[0], 2);
     rotateFilesIfNeeded(); //or rotateFileForIncrementalNumbers();
     qDebug() << "verify that exists " << getCompressedFilename(filenames[0], compressionAlg);
-    QVERIFY(QFileInfo::exists(getCompressedFilename(filenames[0], compressionAlg)));
-    QVERIFY(QFileInfo::exists(filenames[1]));
-    QVERIFY(!QFileInfo::exists(getCompressedFilename(filenames[2], compressionAlg)));
-    QVERIFY(!QFileInfo::exists(getCompressedFilename(filenames[3], compressionAlg)));
+    QVERIFY(QFileInfo::exists(path+getCompressedFilename(filenames[0], compressionAlg)));
```

Comment:
what if `path` does not end with separator? Better to user `getFileWIthFullPath` instead of doing `path + ...`

Reply:
done